style.css

```diff
@@ -1,26 +1,27 @@
 * {
-  margin: 0px;
-  padding: 0px;
+  margin: 0;
+  padding: 0;
 }
 
 body {
-    background-image: url("images/headershapmobil@2x.svg");
-    background-repeat: no-repeat;
-    background-size: cover;
-    width: auto;
-    height: 100%;
-    font-family: 'Poppins', sans-serif;
-    box-sizing: border-box;
+  background-image: url("images/headershapmobil@2x.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: auto;
+  height: 100%;
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+  overflow: scroll;
 }
```

Comment:
[OPTIONAL]

Your background image seems to make this paragraph difficult to read 
![image](https://user-images.githubusercontent.com/78335447/170845177-38f18353-65de-4ded-a9cf-26496e56cf00.png)

I'd advice you to add the background image to the container of your presentation to fix it :wink: